Added New Material System Documentation (#1730)
o3de/o3de.org: content/docs/atom-guide/look-dev/materials/pbr.md

weight: 200
---  

*Physically based rendering (PBR)* is rendering in a photorealistic way by simulating the physics of real-world lighting models. The following sections describe how the concepts of PBR rendering and represented in Atom's core material types.

{{< todo issue="https://github.com/o3de/o3de.org/issues/688" >}}
{{< /todo >}}

## Core PBR Material Types

The following core material types are included in Atom: 

- **StandardPBR**  

>     A full-featured PBR material type, but adopts certain feature limitations in order to improve performance. It provides as much functionality as possible with a limited number of render targets. 

Reviewer: ```suggestion
    A full-featured PBR material type that's optimized by adopting certain feature limitations in order to improve performance. It provides as much functionality as possible with a limited number of render targets. 
```

Author: ```suggestion
    A full-featured PBR material type that provides as much functionality as possible with a limited number of render targets. 
```

Reviewer: Awesome

    
    This file is located in `Gems/Atom/Feature/Common/Assets/Materials/Types/StandardPBR.materialtype`.

- **EnhancedPBR**  
    An enhanced version of StandardPBR that includes additional features, but comes at a higher performance cost. It supports more advanced features that require additional render targets (g-buffers). 

    This file is located in `Gems/Atom/Feature/Common/Assets/Materials/Types/EnhancedPBR.materialtype`. 

> - **BasePBR**  
>     A simplified alternative to StandardPBR that is limited to only the most common PBR features such as *base color*, *metallic*, *roughness*, and *normal*. This is especially helpful for testing and debugging, by eliminating some features from StandardPBR that add considerable complexity.

Author: Your suggestion implies that these four properties are the only ones available, which isn't the case, they are just the most notable ones. Stating "such as" is more accurate.
```suggestion
- **BasePBR**  
    A simplified alternative to StandardPBR that is limited to only the most common PBR features such as *base color*, *metallic*, *roughness*, and *normal*. This is especially helpful for testing and debugging, because it eliminates some features from StandardPBR that add considerable complexity.
```

    This file is located in `Gems/Atom/Feature/Common/Assets/Materials/Types/BasePBR.materialtype`. 
    
- **StandardMultilayerPBR**  

>     Can combine up to three layers of StandardPBR, blended together. Especially useful for breaking up repeated patterns on large surfaces.

Author: Combine->Combines
```suggestion
    Combines up to three layers of StandardPBR, blended together. This is especially useful for breaking up repeated patterns on large surfaces.
```

>     This file is located in `Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR.materialtype`. 

Reviewer: For each material type, link the file location to where it is in the source code. 
Example: 
> This file is located in [`Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR.materialtype`](https://github.com/o3de/o3de/tree/development/Gems/Atom/Feature/Common/Assets/Materials/Types). 

This let's readers see the material type quickly, without needing to locate their local o3de repo.
